staging/src/k8s.io/apiextensions-apiserver/go.mod

```diff
@@ -35,6 +34,7 @@ require (
 	k8s.io/kube-openapi v0.0.0-20241212222426-2c72e554b1e7
 	k8s.io/utils v0.0.0-20241104100929-3ea5e8cea738
 	sigs.k8s.io/json v0.0.0-20241010143419-9aa6b5e7a4b3
+	sigs.k8s.io/randfill v0.0.0-20250303210457-ba7cb2472490
```

Comment:
Follow-up: tag a randfill release?

Reply:
Do we usually tag releases on k-sigs?

Reply:
> Do we usually tag releases on k-sigs?

Yes.

Reply:
Do we have a doc on process?  I have not put tags on kubernetes repos - is it just the usuall github UI ("release") ?

Reply:
We don't, maintainers use different approaches, some projects use tooling like goreleaser, but for a simple library I'd just "git push" a tag and put some notes in the github UI for a release.